### PR TITLE
[FIXED JENKINS-27256] Added functional test for stapler fix

### PR DESCRIPTION
**Pull request by tfennelly**
...  to handle dispatcher methods for getX(long) model methods

See [JENKINS-27256](https://issues.jenkins-ci.org/browse/JENKINS-27256). See https://github.com/stapler/stapler/pull/46

@reviewbybees

### CODE REVIEW

**Comment by stephenc** [comment]
:+1: but cannot merge without a release of stapler

**Comment by jglick** [comment]
:+1: given upstream release.

File: core/pom.xml [hunk 1]
```diff
@@ -39,7 +39,7 @@ THE SOFTWARE.
 
   <properties>
     <staplerFork>true</staplerFork>
-    <stapler.version>1.234</stapler.version>
+    <stapler.version>1.235-20150312.163208-1</stapler.version>
```
Comment (jglick) [inline core/pom.xml]: Need to revert, and add a temporary explicit dep on `org.kohsuke.stapler:stapler:1.235-20150312.163158-1` in this POM.

**Comment by tfennelly** [comment]
Yeah should have marked as a WiP as it deps on the stapler release.

**Comment by daniel-beck** [comment]
Stapler has been released.

**Comment by tfennelly** [comment]
@daniel-beck stapler 1.235 is not in the maven repo yet. Waiting on @kohsuke to do that.

**Pull request merged**